.github/workflows/changeset.yml

```diff
@@ -60,18 +76,56 @@ jobs:
             I see you updated files related to `contracts`. Please run `pnpm changeset` in the `contracts` directory to add a changeset.
           reactions: eyes
           comment_tag: changeset-contracts
+
       - name: Check for new changeset for core
         if: ${{ (steps.files-changed.outputs.core == 'true' || steps.files-changed.outputs.shared == 'true') && steps.files-changed.outputs.core-changeset == 'false' }}
         shell: bash
         run: |
-          echo "Please run pnpm changeset to add a changeset for core."
+          echo "Please run pnpm changeset to add a changeset for core and include in the text at least one tag."
           exit 1
+
       - name: Check for new changeset for contracts
         if: ${{ steps.files-changed.outputs.contracts == 'true' && steps.files-changed.outputs.contracts-changeset == 'false' }}
         shell: bash
         run: |
           echo "Please run pnpm changeset to add a changeset for contracts."
           exit 1
+
+      - name: Check for changeset tags for core
+        id: changeset-tags
+        if: ${{ steps.files-changed.outputs.core-changeset == 'true' }}
+        shell: bash
+        run: bash ./.github/scripts/check-changeset-tags.sh ${{ steps.files-changed.outputs.core-changeset_files }}
+
+      - name: Make a comment
+        uses: thollander/actions-comment-pull-request@fabd468d3a1a0b97feee5f6b9e499eab0dd903f6 # v2.5.0
+        if: ${{ steps.files-changed.outputs.core-changeset == 'true' && steps.changeset-tags.outputs.has_tags == 'false' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          message: |
```

Comment:
I wonder if it's worth setting this message with the tag list one time as a multi-line env var from an init job or early on in this workflow and then reference that to DRY this up?

Reply:
ahh good idea!